docs/Guides/Create-Business-Users.md

```diff
@@ -93,14 +93,14 @@ If another user already exists within your platform with the same email address
 ```
 
 
-## Coming soon
+## Update business locations for an existing user
+You may update a user's accessible business locations in the following way.
 
-### Link an existing user
-You may add one or more add additional business locations to an existing user in the following way.
+After this operation completes, the user will only have access to the specified locations.
```

Comment:
I think you could add this example in addition to the example that was there before instead of replacing it. We could describe the cases as `Add business location(s) to an existing user`, `Replace all business locations for a user` and `Remove business location access from a user`.